config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-presubmits.yaml

```diff
@@ -0,0 +1,19 @@
+presubmits:
+  kubernetes/cloud-provider-gcp:
+  - name: cloud-provider-gcp-tests
+    agent: kubernetes
+    max_concurrency: 5
+    always_run: true
+    context: cloud-provider-gcp-tests
+    rerun_command: "/test cloud-provider-gcp-tests"
+    trigger: "(?m)^/test( all| cloud-provider-gcp-tests),?(\\s+|$)"
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/cloud-builders/bazel
```

Comment:
I really think life will be better if your provide your own image rather than trying to repurpose someone else's image

Looks like this image assumes a /workspace dir exists, which you're not providing in your volume mounts

```console
$ docker run --rm=true --entrypoint=bash gcr.io/cloud-builders/bazel bazel help

FATAL: Output base directory '/workspace/.bazel' could not be created: (error: 13): Permission denied
```

Reply:
Addressed OOB